mcp-test/test-assistant-manual.mjs

```
@@ -0,0 +1,168 @@
+          
+          // If tools were called, execute them and continue
+          if (result.finishReason === 'tool-calls' && result.toolCalls && options.autoExecuteTools) {
+            // Don't add tool calls to messages - AI SDK v5 handles this internally
+            // Just execute the tools and let the next iteration handle it
+            
+            // Execute tools but DON'T add to messages
+            console.log('Executing tools without adding to messages...');
+            for (const toolCall of result.toolCalls) {
+              console.log(`Executing tool: ${toolCall.toolName}`);
+              const tool = options.tools[toolCall.toolName];
+              if (tool && tool.execute) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)